cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -1323,4 +1324,46 @@ public static void create(StructuredTableAdmin tableAdmin) throws IOException {
       createIfNotExists(tableAdmin, IDENTITY_TABLE_SPEC);
     }
   }
+
+  /**
+   * Schemas for operation runs.
+   */
+  public static final class OperationRunsStore {
+    public static final StructuredTableId OPERATION_RUNS =
+        new StructuredTableId("operation_runs");
```

Comment:
lets not mix terminology here. 'run' and 'operation' are synonyms already, let's just name it 'operations'

Reply:
We already have a `Operation` class used in lineage. It also has a corresponding table named `operations_table`. Hence I named the class and the table for LRO as `OperationRun`